entity-service/src/main/kotlin/com/egm/stellio/entity/model/EntityAccessControl.kt

```diff
@@ -0,0 +1,82 @@
+package com.egm.stellio.entity.model
```

Comment:
move it in `authorization` package?

Reply:
Ok I do it

Reply:
This class is more of a model. This place is better here? I can move others in `Auhtorization`?